site/content/learn/headless/basics-playwright-intro.md

```diff
@@ -15,13 +15,14 @@ menu:
 
 The [official documentation](https://playwright.dev/) for Playwright reads:
 
-> Playwright provides a set of APIs to automate Chromium, Firefox and WebKit browsers. By using the Playwright API, you can write JavaScript code to create new browser pages, navigate to URLs and then interact with elements on a page.
-
-The resemblance to Google's [Puppeteer](/learn/headless/basics-puppeteer-intro) is striking, and for good reason.
+> Playwright provides a set of APIs to automate Chromium, Firefox and WebKit browsers. By using the Playwright API, you can write JavaScript code to create new browser pages, navigate to URLs and then interact with elements on a page. 
```

Comment:
Let's mention TypeScript, too. :) 

+ it might be worth sneaking in a note that .net, python etc also work whereas we're only focussing on the JS/TS side of things.